bio/plink/assoc/meta.yaml

```
@@ -0,0 +1,8 @@
+name: plink 
```

This name should be specific to the command, otherwise users will get confused in the menu. So something like:
```suggestion
name: plink_assoc
```